AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/ResNet50_Inference_gpu.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#  Getting Started with [Intel Model Zoo](https://github.com/IntelAI/models)\n",
+    "#  Getting Started with [Intel Reference models](https://github.com/IntelAI/models)\n",
     "\n",
-    "This code sample will serve as a sample use case to perform TensorFlow ResNet50v1.5 inference on a synthetic data implementing a FP32/FP16 and Int8 pre-trained model. The pre-trained model published as part of Intel Model Zoo will be used in this sample. "
+    "This code sample will serve as a sample use case to perform TensorFlow ResNet50v1.5 inference on a synthetic data implementing a FP32/FP16 and Int8 pre-trained model. The pre-trained model published as part of Intel Reference models will be used in this sample. "
```

Comment:
Intel Reference models -> Intel® AI Reference models

Reply:
addressed it accordingly